malt/converters/slices.py

```diff
@@ -52,6 +52,23 @@ def visit_Assign(self, node):
     if replacement is not None:
       return replacement
     return node
+  
+  def visit_Slice(self, node):
```

Comment:
Another option would be to only converted slices that are used in assignments (`x[] = ...`), rather than all slices.

Reply:
I see. Will update this part.